did_core/did_doc/src/schema/service/mod.rs

```diff
@@ -51,6 +47,13 @@ impl Service {
         &self.id
     }
 
+    pub fn contains_service_type(&self, tested_service_type: &ServiceType) -> bool {
+        match &self.service_type {
+            OneOrList::One(service_type) => tested_service_type == service_type,
+            OneOrList::List(service_types) => service_types.contains(tested_service_type),
+        }
+    }
```

Comment:
Removed, users can call in fact rather just call `pub fn service_types(&self) -> Vec<ServiceType>` and call `contains()` on the Vector directly.

I wouldn't rather remove OneOrList from public APIs, rather than promote it.

Reply:
How about
```rust
impl<T> OneOrList<T> {
    pub fn as_slice(&self) -> &[T] {
        match self {
            OneOrList::One(item) => std::slice::from_ref(item),
            OneOrList::List(vec) => vec.as_slice(),
        }
    }
}
```
and then
```
pub fn service_types(&self) -> &[ServiceType]
```
This could be applied to any method returning `OneOrList`...